### DISCUSSION TITLE
Release date of v8?

### DISCUSSION THREAD

**Discussion by geospatialmapmashup** [TerriaJS/terriajs#5130] [Q&A] [answered]
When is the planned release date of v8 Terriajs?

**Comment by AnaBelgun** [comment]
Hi @geospatialmapmashup, you can already access v8 with the next branch https://github.com/TerriaJS/terriajs/tree/next. We are continuing to work on improving the branch and all the new features will be developed only in 'next'. We're releasing new versions of that branch almost every month with new improvements and bug fixes. as we previously did with the master branch.

**Reply by hardreddata** [reply]
Hi @AnaBelgun 

On this and https://github.com/TerriaJS/terriajs/discussions/5134 the TerriaJS page https://github.com/TerriaJS/terriajs/ speaks of a release roadmap for version 8. 

I think a few of us anxiously await this as a way to placate our own stakeholders and plan for the future.

Is this roadmap still in preparation?

Thanks.

**Comment by arky** [comment, marked as answer]
@AnaBelgun @RussellGrew  Is there a tentative release date for v8?

**Reply by hardreddata** [reply]
@arky I am not strictly affiliated with Terria. Still running v7 here. 

I figure once https://nationalmap.gov.au/ switches over it is probably fit for most purposes.

**Reply by arky** [reply]
Thanks @RussellGrew for the tip. I'll keep an eye out.

**Reply by AnaBelgun** [reply]
hi Arky, we're quite close to make the next branch, master. We'll make sure we communicate when we are about to switch.
